pathways/en.yml

```diff
@@ -1,6 +1,45 @@
 ---
 en:
   pathways:
+    ai-toolkit:
+      title: "AI toolkit"
+      subtitle: "Engage with a range of artificial intelligence and machine learning platforms!"
+      description: The AI toolkit introduces various artificial intelligence applications and technologies through hands-on projects using different platforms and tools. Students will work with voice recognition, facial recognition, and other AI technologies, gaining a broader understanding of how AI can be applied in different contexts.
+      meta_title: Project path | AI toolkit for kids | Artificial intelligence for kids | Practical projects using voice and facial recognition
+      meta_description: Introduction to artificial intelligence and machine learning for kids, teenagers and young adults. Build applications that use artificial intelligence with these free resources.
+      header:
+        - title: "What will I create?"
+          content: |
+            Make interactive applications and games that use image classifiers, voice and face recognition. Explore advanced technologies like Adobe Firefly and practical tools for managing Ollama and Stable Diffusion models on Raspberry Pi.
+        - title: "What do I need to know?"
+          content: |
+            - Basic typing and computer navigation skills
+            - Familiarity with our [Introduction to Scratch](https://projects.raspberrypi.org/en/pathways/scratch-intro){:target="_blank"} path may also be useful.
+        - title: "What do I need?"
+          content: |
+            There is a range of projects for learners to try and all projects specify what is required.
+            - One project can be done as an unplugged activity and all printable materials are provided for download.
+            - Most projects require a computer or mobile device capable of running an online coding platform.
+            - Optionally, an account to save projects.
+            - Some projects require a webcam.
+            - Two projects require a Raspberry Pi.
```

Comment:
Probably best to leave off how many projects in case we add more. Maybe change to "Raspberry Pi (optional)" or something like that?